openapi.yaml

```diff
@@ -1162,14 +1123,14 @@ components:
               description: The employee class attached to the job, based on the Employee
                 Class entered on the Position Form (NBAPOSN).
               example: CA
-        employerIdentification:
+        employeeGroup:
           type: object
           properties:
             code:
               type: string
-              nullable: false
-              description: Employer identification code.
-              example: ORST
+              nullable: true
+              enum: [NOSU]
```

Comment:
Null is also accepted and more values may be added in the future. See Illya's comment https://xe.ucsadm.oregonstate.edu:9990/ords/osprod_actwon/actwon.act_servicerequest_main.view_ticket?pSR=59118

Reply:
Ok, so is it only for `POST` or also for `PATCH`?
Right now since you joint `JobPostAttributes` and `JobAttributes` for `JobResource` in [openapi](https://github.com/osu-mist/persons-api/blob/f014c16b7da2118ceae8b47a0e945cbc6a2feaa0/openapi.yaml#L1476-L1477), it makes this field in GET jobs body also only allows `NOSU` and null, which I think is not correct.

Reply:
It is for both post and patch. I fixed it so the enum doesn't show up on the GET body